protobuf_deps.bzl

```diff
@@ -5,6 +5,14 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 def protobuf_deps():
     """Loads common dependencies needed to compile the protobuf library."""
 
+    if not native.existing_rule("zlib"):
```

Comment:
Should this be "bazel_skylib" here instead of "zlib"?

Reply:
Indeed.